static/css/comingsoon.scss

```diff
@@ -0,0 +1,97 @@
+
+/**** BUTTONS ****/
+.stylebuttonblue {
+   background-color: #172B59;
+  color: fff;
+  font-family: 'Source Sans Pro';
+  font-size: 21px;
+  font-weight: 400;
+  padding: 10px;
+  width: 185px; }
+
+
+.stylebuttonblue:hover {
+  background-color: #d3d3d3 !important;
+  color: #172B59 !important;
+  color: fff;
+  font-family: 'Source Sans Pro';
+  font-size: 21px;
+  font-weight: 400;
```

Comment:
repeating code, please be precise as possible, we are using scss you can do nesting